flink-optimizer/src/test/java/org/apache/flink/optimizer/operators/CoGroupGlobalPropertiesCompatibilityTest.java

```diff
@@ -93,7 +95,34 @@ public int partition(Object key, int numPartitions) {
 				GlobalProperties propsRight = new GlobalProperties();
 				propsRight.setCustomPartitioned(keysRight, part);
 				
-				assertTrue(descr.areCompatible(reqLeft, reqRight, propsLeft, propsRight));
+				assertTrue(descr1.areCompatible(reqLeft, reqRight, propsLeft, propsRight));
+			}
+
+			TestDistribution dist1 = new TestDistribution(1);
+			TestDistribution dist2 = new TestDistribution(1);
+			CoGroupDescriptor descr2 = new CoGroupDescriptor(keysLeft, keysRight);
+			
+			// test compatible range partitioning
```

Comment:
We also need tests for a range partitioning with two (or more) keys and tests for range partitioning with different orders.
The tests should check for compatible and incompatible partitionings.
Same applies for the `JoinGlobalPropertiesCompatibilityTest`.